extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -47,14 +47,14 @@ public function awardCustomNotInTrackBadge( $user, $badge_type_id ) {
 		global $wgExternalSharedDB;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgExternalSharedDB" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)